config/routes.rb

```diff
@@ -11,13 +11,17 @@
 
   resources :recipes do
     resources :recipe_ingredients, only: [:edit, :update]
+    resources :reviews, only: [:index, :new, :create, ]
   end
+
+  resources :reviews, only: [:show, :edit, :update, :destroy]
```

Comment:
not sure how having your routes like that makes sens... You will show the recipe that aligns with one recipe